code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -71,6 +71,10 @@ emp_act
 			var/obj/item/clothing/C = bp
 			if(C.body_parts_covered & def_zone.body_part)
 				protection += C.armor[type]
+	if(istype(loc, /obj/mecha))
+		protection += 50
+		if(istype(loc, /obj/mecha/working/hamsandwich))
```

Comment:
Make this a variable or proc on the mech to return the protection instead of hardcoding.